RRS/staff/views.py

```diff
@@ -19,7 +22,17 @@ def create_reservation(request):
 
 def edit_restaurant(request):
     
-    context = {}
+    # TODO(noelsison2): Get current user's associated restaurant
+    restaurant = get_object_or_404(Restaurant, pk=1)
+    
+    if request.method == 'POST':
+        form = RestaurantForm(request.POST, instance=restaurant)
+        if form.is_valid():
+            form.save()
+    else:
+        form = RestaurantForm(instance=restaurant)
```

Comment:
Maybe we can change RestaurantForm(instance=restaurant) to RestaurantForm() so that it is clear that this is an empty form?